analyses/mutational-signatures/05-fit_cns_signatures.R

```diff
@@ -0,0 +1,59 @@
+# Load libraries -------------------------------------------
+library(optparse)
+library(signature.tools.lib) # for getting the CNS signatures
+
+# Set up command line options -------------------------------
+option_list <- list(
+  make_option(c("--abbreviated"),
+              type = "integer",
+              default = 0,
+              help = "Run an abbreviated analysis with fewer iterations?"))
```

Comment:
I might suggest `action = "store_true"` (with `type = "logical"`) here, as that is kind of what I would expect for something called `--abbreviated`, but this opinion is not very strongly held.

Reply:
I ended up keeping this as `integer` since it seems to play poorly with the CI parameter and I didn't fight the fight with type coercion from character too heavily.